### DISCUSSION TITLE
Why my website don't get traffic?

### DISCUSSION THREAD

**Discussion by RajaAbdullah302** [community/community#28027] [Other Feature Feedback, Questions, & Ideas]
I have a website which provide LEGO lights. I don't why I can't get more traffic after working on at least 3 month. If any one of you help me then please let me know. [redacted due to Code of Conduct] visit and tell me my mistakes.

**Comment by a-fat-k79** [comment]
Do you know SEO?

**Reply by jackmarki** [reply]
yes, bro. 
I provide you SEO service if you are interested

**Comment by jfullstackdev** [comment]
being active on social media in every possible way, promoting the link, much more if you can do paid advert

look at my profile, I'm not even promoting it but I'm active here, so that's one effect even without the intention of that 

![image](https://user-images.githubusercontent.com/47092464/182395596-77df9357-0c24-4ec2-b811-d35fd6d0df61.png)

**Comment by queenofcorgis** [comment]
Hi @RajaAbdullah302, we’re glad you’re here 😀

Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by 3kh0** [reply]
@RajaAbdullah302 Please mark this as the answer

**Reply by jackmarki** [reply]
Do your website optimization its need to get a traffic on site.

**Reply by 3kh0** [reply]
Yes
